openrewrite/src/javascript/parser.ts

```diff
@@ -1023,7 +1023,40 @@ export class JavaScriptParserVisitor {
     }
 
     visitConstructSignature(node: ts.ConstructSignatureDeclaration) {
-        return this.visitUnknown(node);
+        return new J.MethodDeclaration(
+            randomId(),
+            this.prefix(node),
+            Markers.EMPTY,
+            [], // no decorators allowed
+            [], // no modifiers allowed
+            node.typeParameters
+                ? new J.TypeParameters(
+                    randomId(),
+                    this.suffix(this.findChildNode(node, ts.SyntaxKind.NewKeyword)!),
```

Comment:
@OlegDokuka, would you prefer to leave this code as is or try to generalise the `mapTypeParametersAsObject` method to accept/handle custom suffixes?

Reply:
lets adjust the `mapTypeParametersAsObject` to fit the current needs